packages/dev/codemods/src/s1-to-s2/src/codemods/codemod.ts

```diff
@@ -203,9 +203,10 @@ export default function transformer(file: FileInfo, api: API, options: Options)
         t.isIdentifier(b.path.node.imported) &&
         (b.path.node.imported.name === 'Item' || b.path.node.imported.name === 'Section')
       ) {
-        // Keep Item and Section imports
-        // TODO: remove if they are unused
-        return;
+        // Keep Item and Section imports if they are still used
+        if (root.find(j.JSXIdentifier, {name: b.path.node.imported.name}).length > 0) {
```

Comment:
Wouldn't this fail if you had another variable in the file that shadowed the declaration?

```js
import {Item} from '@adobe/react-spectrum';

function foo() {
  let Item = 'button';
  // this item shouldn't be replaced.
}
```

Probably should use the scope for this, e.g `b.referenced` should tell you if the binding is referenced. But I think you will need to do a `bindings[0]?.path.scope.crawl()` to re-update the scope. I tried this before but ran into an error.

Reply:
`b.referenced` didn't work, but `b.path.isReferencedIdentifier()` works better but doesn't totally cover us. I'll try to look deeper

Reply:
Should be fixed now, and added a new test